hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("#num");
+elem.addEventListener("input", handleInput);
 
-elem.addEventListener('input', handleInput);
+function checkPalindrome(number) {
+  const num = String(number);
+  const backNum = num.split("").reverse().join("");
+  return num === backNum;
+}
+
+function checkNum(number) {
+  const answer = document.querySelector(".result");
+  answer.classList.remove("text-success", "text-danger");
+
+  if (isNaN(number) || number < 0) {
+    answer.textContent = "Please enter a positive number!";
+    answer.classList.add("text-danger");
+  } else {
+    const valid = checkPalindrome(number);
+
```

Comment:
This is a good idea check both NaN and negative numbers in the same function.

Reply:
Thanks for your feedback.